src/app/dev/platforms/desktop/DevToys.MacOS/Controls/BlazorWebView/BlazorWKWebView.cs

```diff
@@ -10,49 +10,48 @@ namespace DevToys.MacOS.Controls.BlazorWebView;
 // TODO: Credit MAUI repository
 internal sealed partial class BlazorWkWebView : IDisposable
 {
+    private const string DevToysInteropName = "devtoyswebinterop";
```

Comment:
This is a tiny refactoring here. When I tried to make my own GTK wrapper for Linux I had to recode the whole BlazorWebView and learned a bit more about it. In a separate PR, at the occasion, I will refactor both BlazorWebView from MacOS and Linux to have much more code in common.